ngraph/frontend/paddlepaddle/src/op/transpose2.cpp

```diff
@@ -34,7 +34,7 @@ OutputVector transpose2 (const NodeContext& node) {
     PDPD_ASSERT(perm.size() == rank, "transpose2: axis size must equal to data rank!");
 
     auto input_order = ngraph::opset6::Constant::create(ngraph::element::i64, {rank}, perm);
-    return {std::make_shared<ngraph::opset6::Transpose>(data, input_order)};
+    return node.default_single_output_mapping({std::make_shared<ngraph::opset6::Transpose>(data, input_order)}, {"Out"});
```

Comment:
should this be 'opset7' as well? Or it is planned to fix everywhere later?

Reply:
it's better to fix it everywhere, the general recommendation is to use the latest available opset

Reply:
will be done in separate PR